### PR TITLE
Project movies - Therese H & Anna-Kaisa H

### PR DESCRIPTION
**Pull request by ThereseHag**
Link to my deployed project:
https://therese-anna-kaisa-movies.netlify.app/

My reflections on how this week's project turned out:

This week contained a lot of new information to grasp, and we both needed some time to let the info sink it, but the cooperation and pair work has proceeded in a very good way and we've had fun. We started with the structure and then continued with styling. We had some trouble deciding whether to go with displaying 4 movie cards on all screen sizes, or to have 1 for mobile, 2 for tablet and 4 for desktop, but finally decided to go with the later. Working with React still feel more structured and easy to understand than vanilla JS in general. 

Things I'd like to have clarified or explained in more detail:
Nothing in particular, but look forward to learn more about React Router in the future.

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,72 @@
...
+.movie-card {
+  width: 50%;
+}
+}
+
+@media (min-width: 1200px) {
+
+ .movie-card {
+  width: 25%;
+}
```
Comment (dannuzak) [inline code/src/index.css:71]: This part of the code we consider it to be nested and therefore it needs some indentation, in contrast to all the code above. Two spaces indentation according to Technigo guidelines. 🍻

File: code/src/pages/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const { movie_id } = useParams();
+  const [movieDetails, setMovieDetails] = useState({});
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movie_id}?api_key=d54701f7b694b0188ac7819c5b24df16&language=en-US`
+    )
+      .then((response) => response.json())
+      .then((json) => setMovieDetails(json))
+      .catch((err) => console.error(err));
```
Comment (dannuzak) [inline code/src/pages/MovieDetails.js:16]: Those went a bit too far in! 😱  😜

File: code/src/pages/MovieDetails.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import "./moviedetails.css";
+
+export const MovieDetails = () => {
```
Comment (dannuzak) [inline code/src/pages/MovieDetails.js:6]: We think this component is too good to be true! So clean and tidy that Mari-Condo would be so proud of you! 🤓

File: code/src/pages/moviedetails.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
+.button-back {
```
Comment (dannuzak) [inline code/src/pages/moviedetails.css:1]: It is great that you created two css files cause it makes it so easy to read, way to go!🤩

File: code/src/reusables/urls.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+const api_key='d54701f7b694b0188ac7819c5b24df16';
+export const MOVIES_URL = `https://api.themoviedb.org/3/movie/popular?api_key=${api_key}&language=en-US&page=1`;
```
Comment (dannuzak) [inline code/src/reusables/urls.js:2]: A separate variable for the api key is a great idea! We have clever foxes in our foxes team! These two fox-mates are proud of you! 🦊🦊

**Review by dannuzak — APPROVED** [review]
It was a pleasure checking out your code, we followed the same approach and having the opportunity to see how you solve it taught us a better solution. So thanks for that! There was not much to add cause it was really well done...so keep up the great work! We see great things coming on your way for you two...🔮 🤩

File: code/src/components/MovieList.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import { MOVIES_URL } from "../reusables/urls";
+
+export const MovieList = () => {
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetch(MOVIES_URL)
+      .then((response) => response.json())
+      .then((json) => setMovies(json.results))
+      .catch((err) => console.error(err));
```
Comment (KarolCislowski) [inline code/src/components/MovieList.js:14]: Please remember to remove all console outputs from your code before submitting. 👮🏻

File: code/src/pages/MovieDetails.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    <>
+      <div className="main-wrapper">
+        <div
+          className="the-background"
+          style={{
+            backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%), URL(https://image.tmdb.org/t/p/w1280${movieDetails.backdrop_path})`,
+          }}
+          alt={movieDetails.title}
+        >
+          <a href="/" className="button-back">
```
Comment (KarolCislowski) [inline code/src/pages/MovieDetails.js:29]: you should use `<Link to="/">` component provided by the react router. `<a>` restarts your app that should not be happening.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Nice project 🦄  but please remove all console outputs and use `<Link>` component instead of `<a>` elements.

**Review by KarolCislowski — APPROVED** [review]

**Pull request closed without merging**